### DISCUSSION TITLE
SQL Server Support

### DISCUSSION THREAD

**Discussion by don4of4** [collerek/ormar#556] [Q&A]
Based on the first line of the readme, it appears SQL Server is not supported.  That said, given that sqlalchemy core is backing this project, is that really true?  Does this project support SQL Server today?  Very curious since our company uses SQL Server in Azure for most of our RDMS needs.

**Comment by collerek** [comment]
Hi, right now we are limited by the backends supported by databases, as ormar run queries with databases, SQLAlchemy core is used only to construct those queries.
